tests/helpers/zos_blockinfile_helper.py

```diff
@@ -155,3 +183,132 @@ def DsGeneralResultKeyMatchesRegex(test_name, ansible_zos_module, test_env, test
         for key in kwargs:
             assert re.match(kwargs.get(key), result.get(key))
     clean_ds_test_env(test_env["DS_NAME"], hosts)
+
+
+def DsGeneralForce(ansible_zos_module, test_env, test_info, expected):
+    MEMBER_1, MEMBER_2 = "MEM1", "MEM2"
+    hosts = ansible_zos_module
+    if test_env["ENCODING"]:
+        test_info["encoding"] = test_env["ENCODING"]
+    test_info["path"] = DEFAULT_DATA_SET_NAME+"({0})".format(MEMBER_2)
+    try:
+        # set up:
+        # create pdse
+        hosts.all.zos_data_set(name=DEFAULT_DATA_SET_NAME, state="present", type="pdse", replace=True)
+        # add members
+        hosts.all.zos_data_set(
+            batch=[
+                {
+                    "name": DEFAULT_DATA_SET_NAME + "({0})".format(MEMBER_1),
+                    "type": "member",
+                    "state": "present",
+                    "replace": True,
+                },
+                {
+                    "name": DEFAULT_DATA_SET_NAME + "({0})".format(MEMBER_2),
+                    "type": "member",
+                    "state": "present",
+                    "replace": True,
+                },
+            ]
+        )
+        # write memeber to verify cases
+        # print(test_env["TEST_CONT"])
+        hosts.all.shell(cmd="echo \"{0}\" > {1}".format(test_env["TEST_CONT"], test_info["path"]))
+        # copy/compile c program and copy jcl to hold data set lock for n seconds in background(&)
+        hosts.all.zos_copy(content=c_pgm, dest='/tmp/disp_shr/pdse-lock.c', force=True)
+        hosts.all.zos_copy(
+            content=call_c_jcl.format(DEFAULT_DATA_SET_NAME, MEMBER_1),
+            dest='/tmp/disp_shr/call_c_pgm.jcl',
+            force=True
+        )
+        hosts.all.shell(cmd="xlc -o pdse-lock pdse-lock.c", chdir="/tmp/disp_shr/")
+
+        # submit jcl
+        hosts.all.shell(cmd="submit call_c_pgm.jcl", chdir="/tmp/disp_shr/")
+
+        # pause to ensure c code acquires lock
+        time.sleep(5)
+    
+        blockinfile_results = hosts.all.zos_blockinfile(**test_info)
+        for result in blockinfile_results.contacted.values():
+            pprint(result)
+            assert result.get("changed") == 1
```

Comment:
more semantics/style/good practice -- `result.changed` is a bool, so it should get compared to a bool. I know `1` resolves to `true` in python so this technically works, but I'd say this line is better off as:
```
assert result.get("changed") == True
```